seatunnel-e2e/seatunnel-connector-v2-e2e/connector-doris-e2e/src/test/java/org/apache/seatunnel/e2e/connector/doris/DorisIT.java

```diff
@@ -0,0 +1,350 @@
+    private static final String HOST = "doris_e2e";
+    private static final int DOCKER_PORT = 9030;
+    private static final int PORT = 9036;
+    private static final int be_http_port = 8040;
+    private static final int fe_http_port = 8030;
+    private static final String URL = "jdbc:mysql://%s:" + PORT;
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "";
+    private static final String TABLE = "doris_e2e_table";
+    private static final String DRIVER_JAR =
+            "https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.16/mysql-connector-java-8.0.16.jar";
+
+    private static final String SET_SQL =
+            "ADMIN SET FRONTEND CONFIG (\"enable_batch_delete_by_default\" = \"true\")";
+
+    private Connection jdbcConnection;
+
+    private GenericContainer<?> dorisServer;
+
+    private static final String sourceDB = "e2e_source";
+    private static final String sinkDB = "e2e_sink";
+
+    private static final String INIT_DATA_SQL =
+            "insert into "
+                    + sourceDB
+                    + "."
+                    + TABLE
+                    + " (\n"
+                    + "  F_ID,\n"
+                    + "  F_INT,\n"
+                    + "  F_BIGINT,\n"
+                    + "  F_TINYINT,\n"
+                    + "  F_SMALLINT,\n"
+                    + "  F_DECIMAL,\n"
+                    + "  F_LARGEINT,\n"
+                    + "  F_BOOLEAN,\n"
+                    + "  F_DOUBLE,\n"
+                    + "  F_FLOAT,\n"
+                    + "  F_CHAR,\n"
+                    + "  F_VARCHAR_11,\n"
+                    + "  F_STRING,\n"
+                    + "  F_DATETIME_P,\n"
+                    + "  F_DATETIME,\n"
+                    + "  F_DATE\n"
+                    + ")values(\n"
+                    + "\t?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?\n"
+                    + ")";
+
+    private final String COLUMN_STRING =
+            "F_ID, F_INT, F_BIGINT, F_TINYINT, F_SMALLINT, F_DECIMAL, F_LARGEINT, F_BOOLEAN, F_DOUBLE, F_FLOAT, "
+                    + "F_CHAR, F_VARCHAR_11, F_STRING, F_DATETIME_P, F_DATETIME, F_DATE";
+
+    @BeforeAll
+    @Override
+    public void startUp() throws Exception {
+        dorisServer =
+                new GenericContainer<>(DOCKER_IMAGE)
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(HOST)
+                        .withPrivilegedMode(true)
+                        .withLogConsumer(
+                                new Slf4jLogConsumer(DockerLoggerFactory.getLogger(DOCKER_IMAGE)));
+        dorisServer.setPortBindings(
+                Lists.newArrayList(
+                        String.format("%s:%s", PORT, DOCKER_PORT),
+                        String.format("%s:%s", be_http_port, be_http_port),
+                        String.format("%s:%s", fe_http_port, fe_http_port)));
+        Startables.deepStart(Stream.of(dorisServer)).join();
```

Comment:
Please use same doris container in `AbstractDorisIT`

Reply:
ok